MetaMask/metamask-mobile
# refactor: network image util into a selector

## **Description**
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been completely filled out, and PR status checks have passed at least once.


## **Related issues**

Fixes:

## **Manual testing steps**

1. Where the network name was used before, the network name should still display after, there are about 16 places
2. The first half of use cases are used as the currently selected network in the network picker.
3. The other half of use cases are for the network Badge, the first letter of the network name is used as a fallback value to display in the case the image could not be found.

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
